# Define the goal
string roi_id    # Specify a ROI that you want to observe
string[] models  # Specify a ROI that you want to observe
string  mode     # Specify the mode of object search (...)
---
# Define the result
uint32 result
---
# Define a feedback message
string status # PLANNING, READY
float32 percentage_complete

